src/axom/klee/tests/KleeMatchers.hpp

```diff
@@ -85,7 +84,7 @@ class AlmostEqArrMatcher
 
   explicit AlmostEqArrMatcher(const T& arr) : m_arr(arr) { }
 
-  bool MatchAndExplain(const T& other, std::ostream* /* listener */) const
```

Comment:
The changes in this file are related to removing unused parameter names.
Just wanted to highlight the changes here are different from the other files, make sure the style changes in this file are intended.

Reply:
Thanks Brian -- I manually made those changes in a separate commit (https://github.com/LLNL/axom/pull/1125/commits/7a53c7f9ce7bb9187f0a57ee106eeed4d35ea341) to fix some warnings that I introduced in my previous PR (https://github.com/LLNL/axom/pull/1115).  I should have called it out explicitly with a PR comment.